From 10595afb4c007a764b91d28ede5faa00bcb28428 Mon Sep 17 00:00:00 2001
From: Tom Gundersen <teg@jklm.no>
Date: Sat, 4 Oct 2014 13:12:49 +0200
Subject: [PATCH] consoled: move from /bin to /lib/systemd

This should not normally be run manually, but rather through systemd.
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index d2a1767265..3119a95dad 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -3024,7 +3024,7 @@ if ENABLE_TERMINAL
 noinst_LTLIBRARIES += \
 	libsystemd-terminal.la
 
-bin_PROGRAMS += \
+rootlibexec_PROGRAMS += \
 	systemd-consoled
 
 noinst_PROGRAMS += \
